nicegui/client.py

```diff
@@ -320,6 +321,12 @@ def delete(self) -> None:
         self.remove_all_elements()
         self.outbox.stop()
         del Client.instances[self.id]
+        self._deleted = True
+
+    def check_existence(self) -> None:
+        """Check if the client still exists and print a warning if it doesn't."""
+        if self._deleted:
+            log.warning('Client has been deleted but is still being used. This is a bug in the application code.')
```

Comment:
Maybe we should write "This is a bug in your application code". Otherwise the distinction between library and application code is unclear for novice programmers.